db/seeds.rb

```diff
@@ -120,6 +150,10 @@
                   tenant_id: bridge_builders3.id
                   )
 
+debeers_conflict_diamonds_ivorycoast.photos << Photo.create!(
+                  image: File.new( "#{Rails.root}/app/assets/images/debeers_diamonds.jpg")
```

Comment:
Line is too long. [90/80]<br>Indent the first parameter one step more than the previous line.<br>Space inside parentheses detected.